usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -973,10 +936,12 @@ static const_info_t ep11_mechanisms[] = {
     CONSTINFO(CKM_IBM_SHA512_256_KEY_DERIVATION),
     CONSTINFO(CKM_IBM_SHA512_224_KEY_DERIVATION),
     CONSTINFO(CKM_IBM_EC_C25519),
```

Comment:
Yes, because it is used in ep11_get_mechanisms_by_name which itself is used in read_cp_filter_config_file to parse the CP filter config file. If there is an old config filter file that uses the old name, we still want to be able to parse it.